lib/rubocop/cop/factory_bot/association_style.rb

```diff
@@ -158,7 +163,8 @@ def autocorrect_to_implicit_style(corrector, node)
 
         def bad?(node)
           if style == :explicit
-            implicit_association?(node)
+            implicit_association?(node) &&
```

Comment:
Cosmetic. Here it feels that ‘implicit_association?’ can return true when it only looks like an implicit association, not necessarily is.

Reply:
Maybe we should rename the matcher? I can't think of a good name now 🤔